test/labeling/lf/test_nlp_spark.py

```diff
@@ -0,0 +1,47 @@
+import unittest
+from types import SimpleNamespace
+
+import pytest
+from pyspark.sql import Row
+
+from snorkel.labeling.lf.nlp import NLPLabelingFunction
+from snorkel.labeling.lf.nlp_spark import (
+    SparkNLPLabelingFunction,
+    spark_nlp_labeling_function,
+)
+from snorkel.types import DataPoint
+
+
+def has_person_mention(x: DataPoint) -> int:
+    person_ents = [ent for ent in x.doc.ents if ent.label_ == "PERSON"]
+    return 0 if len(person_ents) > 0 else -1
+
+
+@pytest.mark.spark
+class TestNLPLabelingFunction(unittest.TestCase):
+    def _run_lf(self, lf: SparkNLPLabelingFunction) -> None:
```

Comment:
check_person_mention_lf?
_run_lf sounds like it runs an arbitrary lf, but it contains specific checks related to person mentions.